### DISCUSSION TITLE
AI Blog discussion thread for async communication about the project.

### DISCUSSION THREAD

**Discussion by joemcbroom** [joemcbroom/ai-blogs-next#11] [General]
Find a link somewhere to something? Post it here
Have a question? Post it here

**Comment by joemcbroom** [comment]
Added these to the [Reading List](https://github.com/joemcbroom/ai-blogs-next/wiki/Reading-List)
- Supabase Auth with Next.js Server Components
  - https://supabase.com/docs/guides/auth/auth-helpers/nextjs-server-components
- Fetching and caching Supabase data in Next.js 13 Server Components
  - https://supabase.com/blog/fetching-and-caching-supabase-data-in-next-js-server-components

@JulioMoraCode

**Comment by JakobAgat** [comment]
@JulioMoraCode @joemcbroom 
I was messing around with figma and the login screen. What do you guys think? 
![image](https://user-images.githubusercontent.com/122062139/211181147-b9d50f21-6b0b-46f1-aaa4-95bbbef4ddf5.png)
I'm thinking we could have different random cool background images and phrases, just to make it fun/fresh everytime we go in.

**Question:**
Can a regular person land on this admin login page? If so, do we need to show a path back to the public blogverse page? Something like:
- "Not an admin? Go back to the Blogverse" but less serious.

About the logo I'm not set on it yet, but that's the one Alma and I like the most. I want to show you guys 2 options and get your feedback on them. Will post those here too when I have them.

**Reply by joemcbroom** [reply]
I think we will eventually have a login for people to subscribe or comment, stuff like that. so it's good to make it awesome even if we are the only ones looking at it for now. and this is awesome!

**Reply by joemcbroom** [reply]
also it might make sense to incorporate a "register" or create account link. and I'd like to support login with Google, maybe fb and Twitter

**Reply by JakobAgat** [reply]
Woohoo! Glad you like it! I made some changes and added the register link and login with Google, fb and tw. Will work on the admin screens next. Let me know if you have any other comments about the login screen.
<img width="1441" alt="image" src="https://user-images.githubusercontent.com/122062139/211334842-9cdeb3d6-eff9-4125-8c56-9ba3d20b0411.png">

**Comment by joemcbroom** [comment]
```
Write 10 catchphrases for an AI powered blog.  The first would be: "The future is now" 

"The future is now"
"Intelligence amplified"
"Pushing the boundaries of what's possible"
"Where technology meets humanity"
"Empowering progress"
"AI for a better tomorrow"
"Leading the charge in AI innovation"
"Building a smarter world"
"Advancing the way we live and work"
"Bringing intelligence to the forefront"
```

Maybe a couple good ones there 🤷
